dockerfiles/mlrun-api/requirements.txt

```diff
@@ -1,10 +1,11 @@
 uvicorn~=0.27.1
 dask-kubernetes~=0.11.0
-apscheduler~=3.6, !=3.10.2
-sqlite3-to-mysql~=1.4
-objgraph~=3.5
-igz-mgmt~=0.0.10
-humanfriendly~=9.2
+# no support for 4 yet
+# 3.10.2 is bugged for python 3.9
+apscheduler~=3.10,>3.10.2,<4
```

Comment:
do you need to <4 constraint if you have ~=3.10?

Reply:
actually no, it is redundant. I will dismiss and add to test checks